### PR TITLE
#184859055 statistic for seller

### PR DESCRIPTION
**Pull request by nkurunziza1**
**What does this PR do?**
     - Seller should get statistics on his/her sales transaction
**Description of Task to be completed?**
    - The following endpoint must working
    -`GET : /api/v1/stats`
**How should this be manually tested?**
   - Cloning this [repo](https://github.com/atlp-rwanda/destructors-ec-bn.git) , cd into it and run `npm run dev`
   - Using swagger  at this link  `http://localhost:PORT/api-docs` for testing
   - Login as seller  and get Token Then put into `Authorize`
   - Test this endpoints:`/api/v1/stats/`
**What are the relevant pivotal tracker/Trello stories**
   #184859055

### CODE REVIEW

**Review by buka1calvin — COMMENTED** [review]
alexandre your work some of the things are not up to date man like product wishes you still have old ones

**Review by buka1calvin — CHANGES_REQUESTED** [review]
alexander some of apis are not up to date try to double check and update your pr

File: src/controllers/user.controller.js [hunk 1]
```diff
@@ -103,8 +103,7 @@ const loginUser = async (req, res, next) => {
        })
        try{
                await validOTPmail(foundUser,otp,token)
-               return res.status(200).json({message:"please verify your email..."
-              })
+               return res.status(200).json({message:"please verify your email..."})
```
Comment (Abdulkeza) [inline src/controllers/user.controller.js:106]: Do not touch to other's code again

**Comment by nkurunziza1** [comment]
I don't see that problem in my code!

**Pull request merged**